samples/bluetooth/direct_test_mode/src/fem/nrf21540.h

```diff
@@ -53,14 +53,36 @@ enum nrf21540_ant {
 
 /**@brief Initialize nrf21540
  *
- * This function initializes modules needed by the nRF21540 and powers them up.
+ * This function initializes modules needed by the nRF21540 without
+ * powering it up.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int nrf21540_init(void);
+
+/*@brief Power-up nRF21540.
+ *
  * The function is synchronous and waits @ref NRF21540_PD_PG_TIME_US for the
  * device to be activated.
  *
  * @retval 0 If the operation was successful.
  *           Otherwise, a (negative) error code is returned.
  */
-int nrf21540_init(void);
+int nrf21540_power_up(void);
+
+/**@brief Switch nRF21540 to the Power Down state.
+ *
+ * This function switches off the Front End Module. The gain setting
+ * will be reset to the default after calling this function. It can be called
+ * after nrf21540 configuration is cleared by
+ * @ref nrf21540_txrx_configuration_clear. The device will turn off after 10 us.
+ * This function doesn't wait that settling time.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int nrf21540_powerdown(void);
```

Comment:
Shouldn't it be `nrf21540_power_down` for consistency?

Reply:
Can be. I changed it to `nrf21540_power_down`